src/data/rat/defs.lean

```diff
@@ -786,6 +786,13 @@ end casts
 lemma inv_def' {q : ℚ} : q⁻¹ = (q.denom : ℚ) / q.num :=
 by { conv_lhs { rw ←(@num_denom q) }, cases q, simp [div_num_denom] }
 
+protected lemma inv_neg (q : ℚ) : (-q)⁻¹ = -(q⁻¹) :=
+begin
+  simp only [inv_def'],
+  cases eq_or_ne (q.num : ℚ) 0 with hq hq;
+  simp [div_eq_iff, hq]
+end
```

Comment:
Surely we have this for all division rings?

Reply:
`division_ring rat` is in a later file.